app/src/main/kotlin/github/com/rhacco/dota2androidapp/utilities/SharedPreferencesHelper.kt

```diff
@@ -2,15 +2,37 @@ package github.com.rhacco.dota2androidapp.utilities
 
 import android.content.Context
 import android.preference.PreferenceManager
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
 
 class SharedPreferencesHelper(context: Context) {
     private val defaultSharedPreferences = PreferenceManager.getDefaultSharedPreferences(context)
+    private val mSimpleDataFormat = SimpleDateFormat("EEE MMM dd HH:mm:ss z yyyy", Locale.ENGLISH)
 
     fun getFavoriteHero(): String = defaultSharedPreferences.getString(FAV_HERO, "")
 
     fun setFavoriteHero(hero: String) = defaultSharedPreferences.edit().putString(FAV_HERO, hero).apply()
 
+    fun checkProPlayersValid(): Boolean {
+        val validDateString = defaultSharedPreferences.getString(PRO_PLAYERS_VALID_DATE, "")
```

Comment:
Why are we checking date validation for pro players names?

Reply:
To keep the local list of pro players in the database up to date. New pros will be added and I've also seen OpenDota change a registered pro's name once